pom.xml

```diff
@@ -159,6 +164,132 @@
           </testResource>
         </testResources>
         <plugins>
+        <!--
+		    This plugin helps enforce that excluded dependencies stay excluded 
+			and don't sneak back in through another dependency (a global ban). Here are 
+			a few common situations this plugin helps with:
+			
+			1) multiple implementations of a dependency - you only want to use one implementation
+			   (ex: commons-logging vs jcl-over-slf4j, javax.mail vs geronimo-javamail_1.4_spec)
+			2) similar maven artifacts are produced for the same project - you only need to include one 
+			   (ex: cglib vs cglib-nodep)
+			3) an artifact's group id or artifact id has been renamed - maven considers these different
+			   artifacts and you only want to include one
+			   (ex: opensymphony:quartz vs org.opensymphony.quartz:quartz vs org.quartz-scheduler:quartz)
+			4) an artifact was added to the container and no longer is needed
+			   (ex: jaxb in java 6) This plugin is also enforcing maven and jdk versions.
+	      -->
+		  <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-enforcer-plugin</artifactId>
+			<version>1.0-beta-1</version>
+			<executions>
+			  <execution>
+				<id>enforce-banned-dependencies</id>
+				<goals>
+				  <goal>enforce</goal>
+				</goals>
+				<configuration>
+				  <rules>
+					<bannedDependencies>
+						<excludes>
```

Comment:
Claus noticed I had left items in the excludes list commented out.  I have removed all of the items that were commented out due to the differences between Rice 2.6 and Rice 1.0.